services/movie.py

```diff
@@ -0,0 +1,39 @@
+from db.models import Movie
+import init_django_orm  # noqa: F401
+from django.db.models import QuerySet
+
+
+def get_movies(
+        genres_ids: list[int] = None,
+        actors_ids: list[int] = None
+) -> QuerySet:
+    queryset = Movie.objects.all()
+
+    if genres_ids:
+        queryset = queryset.filter(genres__id__in=genres_ids)
+    if actors_ids:
+        queryset = queryset.filter(actors__id__in=actors_ids)
+    return queryset
+
+
+def get_movie_by_id(movie_id: int) -> QuerySet:
+    return Movie.objects.get(id=movie_id)
+
+
+def create_movie(
+        movie_title: str,
+        movie_description: str,
+        genres_ids: list[int] = None,
+        actors_ids: list[int] = None
+) -> QuerySet:
```

Comment:
Are you sure that return type of this function is `QuerySet`?

Reply:
Did you mean class object?

Reply:
Yes, it returns an instance of `Movie` class.